src/components/MarkDownEditor/MarkDownEditor.ts

```diff
@@ -0,0 +1,47 @@
+import MdEditor from 'md-editor-rt';
+import i18n from 'i18n';
```

Comment:
pls add spacing between npm and components import

Reply:
this is correct, they are both from npm.

Reply:
it's not, because we have i18n.ts file in src